docs/product/alerts/create-alerts/metric-alert-config.mdx

```diff
@@ -150,6 +150,9 @@ You can set the status of an alert rule when a threshold is met using the labels
 
 You must set the “Warning” threshold so that it’s triggered before the “Critical” threshold. When Sentry evaluates an alert, the alert’s status is updated to the highest severity trigger that matches. If you don’t set a “Resolved” threshold, the alert automatically resolves when it's no longer breaching the “Critical” or “Warning” conditions. You can also resolve alerts manually.
 
+Note: Anomaly alert thresholds are controlled by Sentry and cannot be manually configured. 
```

Comment:
Should we put this in note format?

Reply:
sure - can you link me to the syntax for that?